Monika After Story/game/screens.rpy

```diff
@@ -2817,31 +2849,55 @@ screen mas_check_scrollable_menu(items, display_area, scroll_align, return_butto
             """
             return {item[0]: item[1]["return_value"] for item in buttons_data.iteritems() if item[1]["return_value"] == item[1]["true_value"] or return_all}
 
-    on "hide" action Function(store.prev_adj.change, 0)
+        def _choose_prompt(buttons_data, selected_prompt):
+            """
+            A method to choose a prompt for the return button.
+            "Nevermind." by default,
+            or selected_prompt if the user has selected something
+
+            IN:
+                buttons_data - the screen buttons data
+                selected_prompt - the prompt for the return button
+
+            OUT:
+                string with prompt
+            """
+            for data in buttons_data.itervalues():
+                if data["return_value"] == data["true_value"]:
+                    return selected_prompt
+            return "Nevermind."
```

Comment:
Interesting that I didn't catch this before, but screens continuously re-run the code inside them. This includes re-running python blocks, and in this case, remaking these functions. Better to just define these elsewhere (in like `mas_ui` probably) and call them here.